src/components/QuickOpenModal.js

```diff
@@ -140,17 +140,16 @@ export class QuickOpenModal extends Component<Props, State> {
       if (selectedSource == null) {
         return;
       }
-      selectLocation({
-        id: selectedSource.get("id"),
-        ...(item.location != null ? { line: item.location.start.line } : null)
-      });
+      const line =
```

Comment:
The 0 part would be covered at component mount time by https://github.com/devtools-html/debugger.html/pull/4836 , I'd rather null this here so as not to have this functionality hardcoded in too many places.